.github/workflows/_python-tests.yml

```diff
@@ -112,9 +112,6 @@ jobs:
         shell: bash
         env:
           PROPERTY_TESTING_PRESET: ${{ inputs.property_testing_preset }}
-      - name: Make get logs script executable
-        if: success() || failure()
-        run: chmod +x bin/get-logs.sh
       - name: Get logs of all services
```

Comment:
did we just not need the chmod

Reply:
yeah, was checked in as executable
copied the new workflow from this area so figured I might as well clean it up